docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/singularity.rst

```diff
@@ -30,16 +34,60 @@ by default in this version of Determined are described below.
 See :doc:`/training/setup-guide/set-environment-images` for the images Docker Hub location and add
 each tagged image required by your environment and experiments to the image cache.
 
-************
- Add Images
-************
+.. _referencing-local-image-paths:
+
+*******************************
+ Referencing Local Image Paths
+*******************************
+
+Each container runtime supports various local container file formats and references them using a
+slightly different syntax. Utilize a cached image by referencing a local path using the experiment
+configuration :ref:`environment.image <exp-environment-image>`.
+
+When using PodMan, you could pull and then save images in OCI archive format to files in a local
+directory ``/shared/containers`` and then reference using the syntax below.
+
+   .. code:: yaml
+
+      environment:
+         image: oci-archive:/shared/containers/cuda-11.3-pytorch-1.10-tf-2.8-gpu
+
+When using Singularity, you could save SIF files in a local directory ``/shared/containers`` and
+then reference them via full path using the syntax below.
+
+   .. code:: yaml
+
+      environment:
+         image: /shared/containers/cuda-11.3-pytorch-1.10-tf-2.8-gpu.sif
+
+Set these ``image`` file references above as the default by specifying them in the
+:ref:`task_container_defaults <master-task-container-defaults>` section of the
+``/etc/determined/master.yaml`` file.
+
+Note: If you specify an image using :ref:`task_container_defaults <master-task-container-defaults>`,
+you prevent new environment container image versions from being adopted on each update of
+Determined.
+
+.. _singularity-image-cache:
+
+*************************************************
+ Configuring a Singularity Image Cache Directory
+*************************************************
+
+When using Singularity, you may use :ref:`referencing-local-image-paths` as described above, or you
+may instead configure a directory tree of images to be searched. To utilize this capability, provide
+a shared directory in :ref:`resource_manager.singularity_image_root <cluster-configuration-slurm>`.
+Whenever an image is referenced, it is translated to a local file path as described in
+:ref:`environment.image <exp-environment-image>`. If found, the local path is substituted in the
+``singularity run`` command to avoid the need for Singularity to download and convert the image for
+each user.
 
-Add each tagged image requried by your environment and the needs of your experiments to the image
+Add each tagged image required by your environment and the needs of your experiments to the image
```

Comment:
"and the needs" doesn't sound right.  Maybe this sentence should read something like:

Add each tagged image needed by your experiments to the image cache: